src/python/pants/backend/core/targets/resources.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, address=None, payload=None, sources=None, **kwargs):
     """
     payload = payload or Payload()
     payload.add_fields({
-      'sources': self.create_sources_field(sources=self.assert_list(sources),
+      'sources': self.create_sources_field(sources=self.assert_list(sources,field_name='sources'),
```

Comment:
pants style (PEP 8) is to use a space after a comma.

Here is the style guide: https://www.python.org/dev/peps/pep-0008/